internal/app_test.go

```diff
@@ -16,213 +12,172 @@ import (
 // Ledger Test Mnemonic: equip will roof matter pink blind book anxiety banner elbow sun young
 
 func Test_FindLedger(t *testing.T) {
```

Comment:
Could maybe also remove the underscores as those seem unidiomatic?

Reply:
Fixed.